packages/browser/src/client/tester/context.ts

```diff
@@ -38,7 +38,15 @@ export function createUserEvent(__tl_user_event_base__?: TestingLibraryUserEvent
     unreleased: [] as string[],
   }
 
-  return {
+  // https://playwright.dev/docs/api/class-keyboard
+  // https://webdriver.io/docs/api/browser/keys/
+  const modifier = provider === `playwright`
+    ? 'ControlOrMeta'
+    : provider === 'webdriverio'
+      ? 'Ctrl'
+      : 'Control'
```

Comment:
I am confused why this is required here. All sent characters are from https://github.com/testing-library/user-event/blob/main/src/keyboard/keyMap.ts shouldn't we do this check in `keyboard`?

We probably need a test for every key 👀

Reply:
> Is this about consistency between providers (like supporting ControlOrMeta both for playwright and webdriverio)?

I think it's about the compatibility with testing-library. If there are keys that are unique to providers, they can stay that way, but common keys should do the same thing and should be entered the same way like in the key map - we can extend the keyMap, by the way

Reply:
Okay, that sounds like a good spec. I'll check later, but providers might only support only `{key}` concept (like `{Control}`), but not physical `[code]` (like `[ControlLeft]`). Is alignment only for `{key}` or you're thinking about to do `[code]` too?

Reply:
I think it's fine to fallback to `Control` when `ControlLeft` is used. There is a `location` property on the even though, so it might be useful for some: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/location

But if there is no way to do that in provider, then we can't do anything about it yet